monkey/infection_monkey/post_breach/signed_script_proxy/windows/signed_script_proxy.py

```diff
@@ -0,0 +1,16 @@
+import subprocess
+
+ORIGINAL_COMSPEC = r'C:\Windows\System32\cmd.exe'
+
+
+def get_windows_commands_to_proxy_execution_using_signed_script():
+    global ORIGINAL_COMSPEC
+    ORIGINAL_COMSPEC = subprocess.check_output('echo %COMSPEC%', shell=True).decode()  # noqa: DUO116
+    return [
+        r'set comspec=infection_monkey\post_breach\signed_script_proxy\windows\random_executable.exe &&',
```

Comment:
While this is Windows-only, I'd still prefer to use Pathlib to manipulate/create paths.

Reply:
What's `random_executable.exe`? How do we compile it? Where's it's source code?

Reply:
It's just a program that outputs "Successfully executed an arbitrary program with the help of a pre-existing signed script on Windows."